### PR TITLE
Feedback edits - Fixing broken links in "on this page" pattern

### PR DESCRIPTION
**Pull request by LanaStewa**
English PR: https://github.com/canada-ca/design-system/pull/336

Note: the broken links in the French request are not parallel to this request.

### CODE REVIEW

File: retroaction/fonctionnement.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ title: Comment fonctionne l’outil de rétroaction sur la page
 ## Sur cette page
 
 * [Pourquoi recueillir la rétroaction des utilisateurs](#pourquoi-recueillir-la-rétroaction-des-utilisateurs)
-* [Conception de l’outil de rétroaction](#conception-de-l-outil-de-retroaction)
-* [Protection des renseignements personnels](#protection-des-renseignements-personnels)
+* [Conception de l’outil de rétroaction](#conception-de-loutil-de-rétroaction)
+* [Protéger des renseignements personnels](#protéger-des-renseignements-personnels)
```
Comment (donohuech) [inline retroaction/fonctionnement.md:23]: This link doesn't work for me. It might be because of the é in it.

File: retroaction/methodes.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ title: Méthodes d’analyse de la rétroaction
 
 * [Qui devrait lire et analyser la rétroaction?](#qui-devrait-lire-et-analyser-la-rétroaction)
 * [Niveau de rétroaction requis](#niveau-de-rétroaction-requis)
-* [Les questions de recherche façonnent l’analyse](#les-questions-de-recherche-faconnent-l-analyse)
-* [Méthodes d’analyse manuelle](#methodes-d-analyse-manuelle)
-* [Outils d’analyse](#outils-d-analyse)
-* [Pilote d’apprentissage automatique](#pilote-d-apprentissage-automatique)
-* [Intégrer d’autres sources de données complémentaires](#integrer-d-autres-sources-de-donnees-complementaires)
+* [Les questions de recherche façonnent l’analyse](#les-questions-de-recherche-faconnent-lanalyse)
```
Comment (donohuech) [inline retroaction/methodes.md:24]: Doesn't work.

File: retroaction/methodes.md [hunk 3]
```diff
@@ -21,11 +21,11 @@ title: Méthodes d’analyse de la rétroaction
...
+* [Méthodes d’analyse manuelle](#methodes-danalyse-manuelle)
```
Comment (donohuech) [inline retroaction/methodes.md:25]: Doesn't work

File: retroaction/methodes.md [hunk 4]
```diff
@@ -21,11 +21,11 @@ title: Méthodes d’analyse de la rétroaction
...
+* [Outils d’analyse](#outils-danalyse)
+* [Pilote d’apprentissage automatique](#pilote-dapprentissage-automatique)
+* [Intégrer d’autres sources de données complémentaires](#integrer-dautres-sources-de-donnees-complementaires)
```
Comment (donohuech) [inline retroaction/methodes.md:28]: Doesn't work

**Review by donohuech — CHANGES_REQUESTED** [review]
A few links don't work. Identified in each line item

**Comment by LanaStewa** [comment]
@donohuech Fixed!

**Review by donohuech — APPROVED** [review]

**Review by prycrane — APPROVED** [review]

**Pull request merged**